### DISCUSSION TITLE
Wiki page id's do not persist after backup and restore.

### DISCUSSION THREAD

**Discussion by Lavhe** [requarks/wiki#7334] [Error / Bug Report]
I have two wikiJS instances

- Instance 1 is where I create all wiki pages and have a storage (push-only) to github
- Instance 2 is a read-only instance it has (pull-only) from the github instance

The issue is that page id's on instance 1 and 2 are not the same and therefore breaking my hyperlinks since I reference them using the page ids

Is this a known thing? and is it an actual issue? is there a workaround?

**Comment by BPowell76** [comment]
Why not use a database backup/dump?

Also, why reference them by page ID? If you use relative url linking (/documents/blah) it will work regardless.
